src/App.jsx

```diff
@@ -14,57 +15,59 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [isSelectedGood, setIsSelectedGood] = useState('Jam');
```

Comment:
Consider initializing `isSelectedGood` with an empty string instead of 'Jam' to avoid having a pre-selected item when the component first renders. This will make the initial state more intuitive.